<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationChooser</class>
 <widget class="QDialog" name="ApplicationChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ApplicationChooser</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="headingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>370</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pick an application for:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="applicationListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>420</width>
     <height>230</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mimetypeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Mimetype</string>
   </property>
  </widget>
  <widget class="QLabel" name="mimetypeIconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Icon</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ApplicationChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ApplicationChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
